conf/app.item.routes

```diff
@@ -109,6 +109,11 @@ POST       /:itemIndex/additional-information/add/:lrn                      cont
 GET        /:itemIndex/additional-information/change-add/:lrn               controllers.item.AddAdditionalInformationYesNoController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = CheckMode, itemIndex: Index)
 POST       /:itemIndex/additional-information/change-add/:lrn               controllers.item.AddAdditionalInformationYesNoController.onSubmit(lrn: LocalReferenceNumber, mode: Mode = CheckMode, itemIndex: Index)
 
+GET        /:itemIndex/consignee/supply-chain-actor/add/:lrn                controllers.item.AddSupplyChainActorYesNoController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = NormalMode, itemIndex: Index)
```

Comment:
Gavin's just removed consignee from the doc, might as well push this through now.